src/button/Button.tsx

```diff
@@ -29,10 +30,18 @@ export interface ButtonProps {
   tabIndex?: number;
 }
 
+const RefButtonComponent = React.forwardRef<HTMLButtonElement, Record<string, any>>(
+  // eslint-disable-next-line prefer-arrow-callback
+  function RefButton(props, ref) {
+    return <button ref={ref} {...props} />;
+  }
+);
+
 function Button({
   testid,
```

Comment:
You need to export the component created from forwardRef 
```suggestion
const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
  // eslint-disable-next-line prefer-arrow-callback
  function ButtonComponent(props, ref) {
    const {testid,  /* ... */ } = props;
    
    // ...
  }
);
```